extensions/amp-story/0.1/amp-story.js

```diff
@@ -585,6 +732,9 @@ export class AmpStory extends AMP.BaseElement {
 
   /** @private */
   preloadPagesByDistance_() {
+    if (this.isDesktop_()) {
+      return;
```

Comment:
Well I think this method uses `transform: translateY()` to force preload by moving the elements close to the viewport so the resource manager will build them. I am not sure why, but the horizontally positioned pages don't seem to need preload forced that way.

I'm not sure if this is related, but occasionally I notice the videos playing before the story page becomes active.

Reply:
Makes sense.  I think we should add an AMP experiment for desktop that gets checked in isDesktop_, so that we can take our time to fix those parts of the experience.  WDYT?

Reply:
That's a good idea, that'll give us a nice safety net

Reply:
Done.